pages/_app.js

```diff
@@ -1,10 +1,11 @@
 import GlobalStyle from "@/components/GlobalStyles";
 import { SWRConfig } from "swr";
 import useLocalStorageState from "use-local-storage-state";
-import "react-toastify/dist/ReactToastify.css";
-import { ToastContainer } from "react-toastify";
+import { ToastContainer, toast } from "react-toastify";
```

Comment:
```suggestion
import { toast } from "react-toastify";
```
You don't seemto be using this :)